app/controllers/api/v1/ecf_participants_controller.rb

```diff
@@ -51,7 +65,10 @@ def lead_provider
       end
 
       def ecf_participant_query
-        ECFParticipants::Index.new(lead_provider, params)
+        Api::V3::ECF::ParticipantsQuery.new(
```

Comment:
This does not look right.
Could we have `Api::V3::ECF::ParticipantsQuery < Api::V1::ECF::ParticipantsQuery`
And use `Api::V3::ECF::ParticipantsQuery` in this class?

Reply:
Hi @StupidCodeFactory tks for your review. I decided to keep the query service in the same V1 module as it's been used in a api/V1 endpoint.. but happy to create `Api::V3::ECF::ParticipantsQuery < Api::V1::ECF::ParticipantsQuery` when we use it also on a api/V3 endpoint. 👍 😄